packages/core/admin/admin/src/content-manager/components/BlocksEditor/index.js

```diff
@@ -43,6 +43,10 @@ const BlocksEditor = React.forwardRef(
         )
       : name;
 
+    const handleBlur = () => {
+      editor.activeSelection = editor.selection;
```

Comment:
Again, no idea why you're doing this, to me it looks redundant but im sure it's not, let's add comments please.